### PR TITLE
Fixed local documentation in line with PR #608 in opta

### PR DESCRIPTION
**Pull request by bigbitbus**
Update the documentation to reflect the functionality of https://github.com/run-x/opta/pull/608

### CODE REVIEW

**Review by RemyDeWolf — COMMENTED** [review]
something seems to be broken for me:

I started from scratch - having no k8s config. You can also delete your ~/.kube/config as well
```
# this file doesn't exist
export KUBECONFIG=$HOME/.kube/opta-local
```

```
opta apply --local -c hello.yaml --auto-approve

Apply complete! Resources: 8 added, 0 changed, 0 destroyed.

Outputs:

k8s_cluster_name = "kind-opta-local-cluster"
public_nat_ips = tolist([
  "75.46.138.82",
])
Opta updates complete!
ERROR: Command '['kubectl', 'config', 'use-context', 'kind-opta-local-cluster']' returned non-zero exit status 1.
Traceback (most recent call last):
  File "/Users/remy/github/opta/opta/cli.py", line 73, in <module>
    cli()
  File "/Users/remy/github/opta/.venv/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/Users/remy/github/opta/.venv/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/Users/remy/github/opta/.venv/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/remy/github/opta/.venv/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/remy/github/opta/.venv/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/Users/remy/github/opta/opta/commands/apply.py", line 101, in apply
    _apply(
  File "/Users/remy/github/opta/opta/commands/apply.py", line 303, in _apply
    raise e
  File "/Users/remy/github/opta/opta/commands/apply.py", line 190, in _apply
    for module_idx, current_modules, total_block_count in gen(
  File "/Users/remy/github/opta/opta/core/generator.py", line 50, in gen
    set_kube_config(layer)
  File "/Users/remy/github/opta/opta/core/kubernetes.py", line 83, in set_kube_config
    _local_set_kube_config(layer)
  File "/Users/remy/github/opta/opta/core/kubernetes.py", line 130, in _local_set_kube_config
    nice_run(
  File "/Users/remy/github/opta/opta/nice_subprocess.py", line 128, in nice_run
    raise CalledProcessError(
subprocess.CalledProcessError: Command '['kubectl', 'config', 'use-context', 'kind-opta-local-cluster']' returned non-zero exit status 1.

```

the kube config file was not created.
```
kubectl config use-context kind-opta-local-cluster 
W0204 11:10:58.125854   98057 loader.go:221] Config not found: /Users/remy/.kube/opta-local
error: no context exists with the name: "kind-opta-local-cluster"
```

File: content/en/Getting started/local.md [hunk 1]
```diff
@@ -135,11 +111,22 @@ You can test your service by visiting [http://localhost:8080/hello](http://local
...
 ```
 
+- View container logs of your application
+```
+opta logs --local -c hello.yaml
+```
+
+- View Kubernetes for your Application's deployment Events
+```
+opta events --local -c hello.yaml
```
Comment (RemyDeWolf) [inline content/en/Getting started/local.md:126]: this command is currently hidden, do we want to make it visible then?

Comment (bigbitbus) [reply content/en/Getting started/local.md:126]: It works great so no reason to hide it imo, but @NitinAgg do you have another opinion?

File: content/en/Getting started/local.md [hunk 2]
```diff
@@ -37,29 +38,22 @@ Running Opta Local needs a reasonably powerful PC. These requirements are not fo
...
+Running your application in Opta Local environment is almost identical to how you would run them on public cloud using the Opta CLI. We show a couple of opta yaml examples to get you started.
 
 #### Example 1: Deploy a service using an existing docker image
 
 We will deploy a simple [hello app](https://github.com/run-x/opta-examples/tree/main/hello-app) by defining these files.
 
-{{< tabs tabTotal="2" tabID="1" tabName1="hello.yaml" tabName2="local.yaml" >}}
+{{< tabs tabTotal="1" tabID="1" tabName1="hello.yaml" tabName2="local.yaml" >}}
 {{< tab tabNum="1" >}}
 {{< highlight yaml "hl_lines=11" >}}
```
[the remark this answers is not in the document]
Comment (bigbitbus) [reply content/en/Getting started/local.md]: Done

File: content/en/Getting started/local.md [hunk 3]
```diff
@@ -160,32 +143,31 @@ NAME                                                          REFERENCE
...
     - `hello.yaml` The service Opta file. 
-    - `local.yaml` The environment Opta file. 
     - `app.py` The application code.
     - `Dockerfile` The docker file to build the application.
-1. Set `image: AUTO` in `opta.yaml` to use the local image.
-1. Make a change to the returned text in `app.py` to validate that the new image is used.
+2. Set `image: AUTO` in `hello.yaml` to use the local image.
+3. Make a change to the returned text in `app.py` to validate that the new image is used.
 
 {{< tabs tabTotal="4" tabID="2" tabName1="opta.yaml" tabName2="local.yaml" tabName3="app.py" tabName4="Dockerfile" >}}
```
Comment (RemyDeWolf) [inline content/en/Getting started/local.md:165]: opta.yaml -> hello.yaml

**Review by RemyDeWolf — APPROVED** [review]
one more tiny comment but overall looks and works great

File: content/en/Getting started/local.md (hunk 3 above)
Comment (RemyDeWolf) [reply content/en/Getting started/local.md:165]: also remove `tabName2="local.yaml" ` from this and renumber the names

Comment (bigbitbus) [reply content/en/Getting started/local.md]: Fixed

**Pull request merged**